client/my-sites/site-settings/analytics/form-google-analytics-simple.jsx

```diff
@@ -73,9 +75,11 @@ const GoogleAnalyticsSimpleForm = ( {
 	};
 
 	const renderForm = () => {
-		const plan = findFirstSimilarPlanKey( site.plan.product_slug, {
-			type: TYPE_PREMIUM,
-		} );
+		const plan = ! isSEOEligible
```

Comment:
This also confuses me. What's the connection between the SEO and the GA features?

Reply:
This is a check for which plan slug to use in the URL (use `value-bundle` for free Atomic). We could change it to an Analytics eligibility check.

Reply:
Since the plan slug actually depends on the site (`value-bundle` for WP.com sites, `jetpack_security_daily_monthly` for Jetpack sites) and not on the feature, I'd use a site check here. I think that's less confusing.

Reply:
Thanks, I've changed to checking for a WPCOM site with a free plan here. Otherwise `findFirstSimilarPlanKey()` returns a Jetpack plan. 

For all other WPCOM plans we're checking the current plan duration to determine which slug to use.